src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs

```diff
@@ -636,9 +636,16 @@ public ProcessResult EmailMultiplePurchaseOrders(IList<int> orderNumbers, int us
                 else
                 {
                     var html = this.purchaseOrderTemplateService.GetHtml(order).Result;
-                    this.SendOrderPdfEmail(html, supplierContactEmail, copyToSelf, userNumber, order, null);
-                    text += $"Order {orderNumber} emailed successfully to {supplierContactEmail}\n";
-                    success++;
+                    try
+                    {
+                        this.SendOrderPdfEmail(html, supplierContactEmail, copyToSelf, userNumber, order, null);
+                        text += $"Order {orderNumber} emailed successfully to {supplierContactEmail}\n";
+                        success++;
+                    }
+                    catch (MimeKit.ParseException exception)
+                    {
+                        text += $"Order {orderNumber} to {supplierContactEmail} failed. {exception.Message}. \n";
+                    }
```

Comment:
you could also just add a generic Exception catch after this more specific one in case any unforseen exceptions raise their heads?

Reply:
good idea. i was swithering whether to catch more than that